docs/modules/ROOT/pages/metrics-details.adoc

```diff
@@ -1,30 +1,28 @@
 
-:root-target: ./../../build/documentedMetrics/
-
-### Meters and tags for Reactor-Core-Micrometer module
+= Meters and tags for Reactor-Core-Micrometer module
```

Comment:
This part renders inappropriately as a top-level header inside the body of the metrics.adoc. It should be at the same level as `Scheduler metrics` and `Publisher metrics` headers.

Reply:
This issue is also causing the lower-level headers such as `Micrometer.metrics()` and their descendants to be a level higher than they should. Please consider whether they can be properly adjusted or what are the options if not.

Reply:
I think it's fixed in https://github.com/reactor/reactor-core/pull/3806/commits/fad1622c1da5ab72ea90732c09a8d49f5b421489

Reply:
Yes, thank you, now it looks ok.